service/api/cadence/client.go

```diff
@@ -36,10 +36,16 @@ func (t *cadenceClient) Close() {
 }
 
 func (t *cadenceClient) StartInterpreterWorkflow(ctx context.Context, options api.StartWorkflowOptions, args ...interface{}) (runId string, err error) {
+	workflowIdReusePolicy, err := mapToTemporalWorkflowIdReusePolicy(options.WorkflowIDReusePolicy)
+	if err != nil {
+		return "", nil
+	}
 	workflowOptions := client.StartWorkflowOptions{
 		ID:                           options.ID,
 		TaskList:                     options.TaskQueue,
 		ExecutionStartToCloseTimeout: options.WorkflowRunTimeout,
+		WorkflowIDReusePolicy:        workflowIdReusePolicy,
+		CronSchedule:                 options.CronSchedule,
```

Comment:
@longquanzheng do we need validate cron schedule here?

Reply:
I think the Cadence/Temporal go SDK will do it so we don't need to do here